broskichat-app/src/App.css

```diff
@@ -14,6 +14,7 @@
   color: white;
 }
 
+
```

Comment:
@mattkim8 we should be careful to not introduce extra spaces when merging. This file is going to be deleted though so ndb. Just a good way to illustrate something 👍